engine/client/gl_local.h

```diff
@@ -349,6 +349,72 @@ void R_LightForPoint( const vec3_t point, color24 *ambientLight, qboolean invLig
 int R_CountSurfaceDlights( msurface_t *surf );
 int R_CountDlights( void );
 
+
+/// <STROBE> // Move to strobe.h ?
+//#define SolidConsoleX 0
+//#define SolidConsoleY (y - scr_width->value * 3 / 4)
+//#define SolidConsoleW (scr_width->value)
+//#define SolidConsoleH (scr_width->value * 3 / 4)
+//extern wrect_t con_rect; //Float - Int incompatibility
+typedef struct _rectf_s
+{
+	float x, y, w, h;
+} _rectf_t;
+extern _rectf_t con_rect;
+
+void R_Strobe( void );
+
+typedef enum {
+	p_positive = 1, // Phase: Positive
```

Comment:
We have BIT() macro for eaiser bitmasks.

Reply:
Marked. Do we have more generic handling for flags or is it only this BIT macro?

Reply:
No, it's only BIT macro. 

Unkle Mike refactored it. He added FBitSet() for bit checking, ClearFlag, SetFlag and ResetFlag macros. But fork still don't moved on his version.